### PR TITLE
Sanitize url generated from response class

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes : #685

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

### Preview Link 

- **Replace XXX with your PR no**
- Link to live demo: http://pr-XXX-fossasia-badgeyay.surge.sh  

#### Changes proposed in this pull request:

- Sanitize url generated from the backend
-

### CODE REVIEW

**Comment by yashLadha** [comment]
Ping @djmgit @ParthS007 @gabru-md @vaibhavsingh97 Please review it.

File: backend/blueprint/api/config/urlConfig.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Configuration variable for the application
+BASE_URL = 'http://badgeyay-api.herokuapp.com/'
```
Comment (djmgit) [inline backend/blueprint/api/config/urlConfig.py]: When we are testing locally, we will always have to change this to localhost, and again before commiting we have to change it back.
On other hand, if we hard code it to localhost here, auto deployment will cause trouble.
Do we have any other alternative here?

Comment (yashLadha) [reply backend/blueprint/api/config/urlConfig.py]: I will try to use the debug config as locally we will set it as true but not in the production. What do you think?

Comment (djmgit) [reply backend/blueprint/api/config/urlConfig.py]: yes sounds good :+1:

File: backend/blueprint/api/utils/response.py [hunk 2]
```diff
@@ -12,8 +14,12 @@ def generateMessage(self, message):
     def generateURL(self, url, message):
         self.message = message
         self.URL = url
+        self.sanitizeURL()
         return self.serialize()
```
Comment (gabru-md) [inline backend/blueprint/api/utils/response.py]: shift `self.serialize()` inside `self.sanitizeURL()` maybe?
:sweat_smile:  what say?

Comment (gabru-md) [reply backend/blueprint/api/utils/response.py]: maybe I am talking about something like 
```python
return self.saniitizeURL()
```

where sanitize has declaration as:
```python
def sanitizeURL(self):
    self.URL = #change the URL
    return self.serialize()
```

But it is okay, your approach is great. I was just suggesting a minor change :sweat_smile: :+1:

Comment (yashLadha) [reply backend/blueprint/api/utils/response.py]: Sure that i can do :+1:

**Comment by yashLadha** [comment]
Made changes @gabru-md  Thanks for the review. Please review it again @djmgit @ParthS007 @vaibhavsingh97

**Comment by yashLadha** [comment]
May be we can update in readme to copy that file to some other name and it to `.gitignore`. Like copying a tempfile to `config.py` and adding the same to `.gitgnore` will help to resolve both of our issues.

**Comment by yashLadha** [comment]
`cp config/dev_config.py config/config.py`

**Pull request merged**